app/graphql/types/inputs/delete_input.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Types
+  module Inputs
+    class DeleteInput < ::Types::Base::InputObject
+      I18N_PATH = 'graphql.inputs.delete_input'
+
+      graphql_name 'DeleteInput'
+
+      description I18n.t("#{I18N_PATH}.desc")
+
+      argument :id,
+               ID,
+               required: true,
+               description: I18n.t("#{I18N_PATH}.args.id"),
+               prepare: ->(id, _ctx) { id.to_i }
```

Comment:
I think that your code will work the same without this prepare lambda)

Reply:
And if you decide to switch from incremental id to UUID, this won't work at all. I would delete it

Reply:
Agree, done